src/types/onyx/ReimbursementAccountDraft.ts

```diff
@@ -20,7 +23,7 @@ type CompanyStepProps = {
     website?: string;
     companyTaxID?: string;
     incorporationType?: string;
-    incorporationDate?: string | Date;
+    incorporationDate?: string;
```

Comment:
Why are we loosening the type here?

Reply:
as far as I remember when checking this, the actual output of the date picker is string. Date is used within it tho.

Reply:
ok thanks!